release-notes/unreleased/2370-fix-ios-crash-calendar-export.yml

```diff
@@ -0,0 +1,6 @@
+issue_key: 2370
+show_in_stores: true
+platforms:
+  - ios
+en: Exporting an event to the calendar app works again
+de: Der Eventexport funktioniert wieder
```

Comment:
I think this never did not work, did it? I think only viewing the event in the calendar didn't.

Reply:
True, ooops